build.gradle

```diff
@@ -65,13 +75,13 @@ dependencies {
     api 'androidx.multidex:multidex:2.0.1'
 
     //Kotlin reflect
-    api "org.jetbrains.kotlin:kotlin-reflect:1.4.32" //TODO: what does this thing do??
+    api "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version" //TODO: what does this thing do??
 
     //ViewModel
-    api "androidx.lifecycle:lifecycle-extensions:2.2.0"
-    api "androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1"
+    api "androidx.lifecycle:lifecycle-extensions:2.2.0" // TODO: This is deprecated (https://developer.android.com/jetpack/androidx/releases/lifecycle#2.2.0)
```

Comment:
I actually think this one can be removed without any issues...

The complete list of possible `lifecycle dependencies` can be found [here](https://developer.android.com/jetpack/androidx/releases/lifecycle#declaring_dependencies)
With that list in mind, the only extensions we are using are for the ViewModels and LiveData.